lua/ulx/modules/sh/cfc_curse.lua

```diff
@@ -9,6 +9,36 @@ do -- Load curse effects
 end
 
 
+local function getDurationStrings( durationSeconds, effectOverride )
+    local hasCustomDuration = durationSeconds > 0 and ( not effectOverride or not effectOverride.blockCustomDuration )
+    local durationAppend = ""
+    local briefly = " briefly"
+
+    if hasCustomDuration then
+        local durationStr = durationSeconds >= 60 and
+            ULib.secondsToStringTime( durationSeconds ) or
+            math.Round( durationSeconds ) .. " seconds"
+
+        durationAppend = " for " .. durationStr
+        briefly = ""
+    end
+
+    return durationAppend, briefly
+end
```

Comment:
When no duration is supplied, the command prints out *"... briefly cursed"* instead of *"... cursed with ... for DURATION"*. This existed as a code snippet previously, but has been moved out into a helper function so it can be re-used for amount-based printouts.

The ulx curse command code is (and was) a big tech-debt-y mess regardless, but this'll at least help with DRY.